network.js

```diff
@@ -1,3 +1,5 @@
+require('dotenv').config()
```

Comment:
why did you add this?

Reply:
this is from a previous state, trying to use Gaia Testnet from the Figment's datahub (it needs an API key)